tests/integration/components/daily-calendar-event-test.js

```diff
@@ -3,24 +3,24 @@ import { setupRenderingTest } from 'dummy/tests/helpers';
 import { setupIntl } from 'ember-intl/test-support';
 import { render } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
-const s = '[data-test-daily-calendar-event]';
 import { setupMirage } from 'ember-cli-mirage/test-support';
 import { DateTime } from 'luxon';
+import { component } from 'ilios-common/page-objects/components/daily-calendar-event';
 
 module('Integration | Component | daily-calendar-event', function (hooks) {
   setupRenderingTest(hooks);
   setupIntl(hooks, 'en-us');
   setupMirage(hooks);
 
-  this.createEvent = function (startDate, endDate) {
-    const lastModified = '2012-01-09 08:00:00';
+  this.createEvent = function (startDate, endDate, lastModified, isScheduled, isPublished) {
     const color = '#00cc65';
-
     this.server.create('userevent', {
-      startDate: DateTime.fromFormat(startDate, 'yyyy-LL-dd hh:mm:ss').toJSDate(),
-      endDate: DateTime.fromFormat(endDate, 'yyyy-LL-dd hh:mm:ss').toJSDate(),
+      startDate: DateTime.fromFormat(startDate, 'yyyy-LL-dd hh:mm:ss').toISO(),
```

Comment:
Just looked at this again, we're using a javascript date for this everywhere else. Any reason for the change here?

Reply:
updated :heavy_check_mark: